Gemfile

```diff
@@ -54,6 +54,8 @@ group :test do
   gem 'capybara' # Use system testing [https://guides.rubyonrails.org/testing.html#system-testing]
   gem 'selenium-webdriver'
   gem 'webdrivers'
+  gem 'rails-controller-testing'
```

Comment:
I wonder if this is needed? The RSpec team recommends not to use controller specs and the rails-controller-testing gem for apps newer than Rails 5:

- [StackOverflow question about Controller specs vs Requests specs](https://stackoverflow.com/questions/40851705/controller-specs-vs-request-specs#:~:text=Indeed%2C%20the%20RSpec%20team%20officially%20states%20controller%20specs%20are%20now%20obsolete.&text=For%20new%20Rails%20apps%3A%20we,to%20write%20request%20specs%20instead.)

- [The RSpec page where they talk about deprecating Controller specs](http://rspec.info/blog/2016/07/rspec-3-5-has-been-released/)